kaldb/src/test/java/com/slack/kaldb/testlib/MessageUtil.java

```diff
@@ -169,9 +170,60 @@ public static List<LogMessage> makeMessagesWithTimeDifference(
     return result;
   }
 
-  // TODO: Move this to TestKafkaServer class.
-  public int getPort() throws IOException {
-    ServerSocket socket = new ServerSocket(0);
-    return socket.getLocalPort();
+  public static List<Trace.Span> makeMessagesWithTimeDifferenceTS(
```

Comment:
by the end of the next PR all tests will start using this and we'll be able to remove `makeMessagesWithTimeDifference` and rename this method